docker-compose.yml

```diff
@@ -1,492 +1,821 @@
 version: "3.7"
 services:
   db:
-    image: mariadb:latest
+    image: mysql:5.7.40
     restart: always
+    secrets:
+      - database_password
     environment:
-      MYSQL_ROOT_PASSWORD: tgbdwf3h
```

Comment:
Git is nasty, and will keep this in memory forever. Might want to delete this commit and just commit it all again without this :) in any way, do not use this password, anywhere, ever again :)

Reply:
The password being obfuscated via BITWISE XOR makes this comment completely irrelevant. That and Daniel put it in a "mysqlpassword" file again. The idea here is that there is a default environment variable that can immediately allow for docker compose to bring up all of the game servers. If it's desired, the operator can change the environment variable to their liking.

Reply:
I eventually would like to draft a pull request that can optionally leverage certificates for even better security